client/components/NavLink.jsx

```diff
@@ -1,6 +1,8 @@
 import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
 import {Link, NavLink} from 'react-router-dom';
-import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+import Index from '../index.jsx';
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter, ButtonDropdown, DropdownToggle, DropdownItem, DropdownMenu } from 'reactstrap';
```

Comment:
So when you are writing import statements. (same for every language), you should group them and order them by alphabet.
like react-dom, react-router-dom are different from ../index.jsx, as index is something you wrote and the others are libraries.
Also, some has space after { while some do not. Some even has {} while some do not.
Please reorder the import statements 

```
import { React, Component } from 'react';
import { Button, Modal, ModalHeader, ModalBody, ModalFooter, ButtonDropdown, DropdownToggle, DropdownItem, DropdownMenu } from 'reactstrap';
import ReactDOM from 'react-dom';
import { Link, NavLink } from 'react-router-dom';
import Index from '../index.jsx';
```